### PR TITLE
[eclipse/xtext-maven#111] Add testGenerate maven goal entry

### PR DESCRIPTION
**Pull request by jrosczak**

### CODE REVIEW

File: xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ Xtext 2.21.0 is mainly a maintenance release.
 
 ## Eclipse Integration
 
+## xtext-maven-plugin
+
+### testGenerate Goal
+
+The maven plugin has now two goals, generate and testGenerate. The new testGenerate goal runs during the generate-test-sources phase and uses the test class path of the maven project.
```
Comment (kthoms) [inline xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md]: maven => Maven
class path => classpath
Please make the goal names `monotype`
`generate`, `testGenerate`, `generate-test-sources`

**Comment by cdietrich** [comment]
@jrosczak we had some problems with the website-master branch and had to restore/force push it.
can you please recreate your PR/commit on top of the current website-master branch

File: xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md (hunk 1 above)
Comment (jrosczak) [reply xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md]: I changed this.

**Comment by cdietrich** [comment]
@jrosczak could you please create a new branch with just your commit on top and update this PR or create a new one (i just want to be 1000% sure that we can merge both ways in the future)
sorry for the inconvenice. @miklossy owes you a beer.

**Comment by jrosczak** [comment]
@cdietrich No problem :-)

**Comment by cdietrich** [comment]
thx @jrosczak

**Pull request merged**